components/shared/button/Button.tsx

```diff
@@ -1,23 +1,42 @@
 import React from 'react';
 import clsx from 'clsx';
 
-export enum ButtonType {
+export enum ButtonLayout {
   Default = 'DEFAULT',
   Black = 'BLACK',
 }
+
+export enum ButtonSize {
+  Small = 'SMALL',
+  Medium = 'MEDIUM',
+  Big = 'BIG',
+}
 interface ButtonProps {
   children: string;
-  type?: ButtonType;
+  type: JSX.IntrinsicElements['button']['type'];
```

Comment:
type: string;

Reply:
mogę dać albo: `'button' | 'submit' | 'reset' | undefined` albo tak jak jest... Sam `string` wywala: `Type 'string' is not assignable to type '"button" | "submit" | "reset" | undefined'.`

Reply:
Zrobiłem finalnie: `type?: 'button' | 'submit' | 'reset';`